docassemble/AssemblyLine/data/questions/al_visual.yml

```diff
@@ -79,18 +79,25 @@ code: |
     {"url": url_ask(['al_start_over_confirmation','al_start_over']),
     "label": "Start over"
     },
+  ]
```

Comment:
This works, but only on brand new interviews. I imagine if someone decided to turn it on they'd want it to apply to all existing interviews.

I never know the best solution for problems like this though. I guess `reconsider: al_menu_items`?

Reply:
Yeah. Making the list is fine, tbh. Most expensive thing is probably just accessing the config probably.

Reply:
I'll put this on the block that computes the value of `al_menu_items` instead, because someone might not want to use our menu items at all.